add_watermark_pdf/script.py

```diff
@@ -0,0 +1,25 @@
+"""
+@hardik
+@2020
+@add_watermark to pdf
+@py code
+"""
+import PyPDF2
+
+inputFile = open('Demo.pdf', 'rb')
+pdfReader = PyPDF2.PdfFileReader(inputFile)
+inputFirstPage = pdfReader.getPage(0)
+
+pdfWatermarkReader = PyPDF2.PdfFileReader(open('watermark.pdf', 'rb'))
+inputFirstPage.mergePage(pdfWatermarkReader.getPage(0))
+pdfWriter = PyPDF2.PdfFileWriter()
+pdfWriter.addPage(inputFirstPage)
+
+for pageNum in range(1, pdfReader.numPages):
```

Comment:
It's already dealing with every page of multipage PDF

Reply:
Agreed but when it comes to image it become tedious.